lib/fluent/plugin/out_irc.rb

```diff
@@ -43,6 +45,8 @@ class IRCOutput < Fluent::Output
       define_method("log") { $log }
     end
 
+    attr_reader :conn # for test
```

Comment:
How about defining a singleton method in test method like below instead of this public getter for only test purpose?